Models/Menu.cs

```diff
@@ -17,8 +17,8 @@ public class Menu
 
         public override string ToString()
         {
-            return $"Tarih:{Date.ToString()}\nÖğlen Yemeği:\n{string.Join("\n", Lunch)}\nAkşam Yemeği:\n{string.Join("\n", Dinner)}";
+            return $"Tarih:{Date.ToString()}\n11.30 - 14.30\nÖğlen Yemeği:\n{string.Join("\n", Lunch)}\n\n16.30 - 19.15\nAkşam Yemeği:\n{string.Join("\n", Dinner)}";
```

Comment:
Saatler **hh:mm** formatında olmalı

Reply:
@mustafacavusoglu PR'ın için teşekkür ederim. Saatleri belirtilen formatta günceller misin?